Documentation/ApiOverview/Seo/ExampleSetInController/_SomeController.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+use MyVendor\MyExtension\PageTitle\MyOwnPageTitleProvider;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+$titleProvider = GeneralUtility::makeInstance(MyOwnPageTitleProvider::class);
```

Comment:
Please avoid using GU:mI anymore. We should show best practise and as such use DI.

The title provider will mostly be used in controller (with or without Extbase), so it can be injected into the constructor.

Reply:
I did not write this code but move it. From where it was embedded in Rst.

Reply:
Hmm, okay, will provide a separate PR after this is merged.